packages/main/src/webComponents/Token/index.tsx

```diff
@@ -22,13 +22,12 @@ export interface TokenPropTypes extends Omit<WithWebComponentPropTypes, 'onSelec
   /**
    * Fired when the a <code>Token</code> is selected by user interaction with mouse or clicking space.
    */
-  onSelect?: (event: CustomEvent<{}>) => void;
+  onSelect?: (event: CustomEvent) => void;
 }
 
 /**
- * Tokens are small items of information (similar to tags) that mainly serve to visualize previously selected items.
-
- * <a href="https://sap.github.io/ui5-webcomponents/playground/components/Token" target="_blank">UI5 Web Components Playground</a>
+ * Tokens are small items of information (similar to tags) that mainly serve to visualize previously selected items.<br />
+ * __Note:__ Tokens shouldn't be implemented as a standalone component. Use the `Tokenizer` to achieve the intended functionality.
```

Comment:
I would change this line. Either remove it or reference the `MultiInput` instead.